Platform/settings/base.py

```diff
@@ -25,6 +25,21 @@
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = os.environ.get("SECRET_KEY", "o7ql0!vuk0%rgrh9p2bihq#pege$qqlm@zo#8&t==%&za33m*2")
+IDENTITY_RSA_PRIVATE_KEY = os.environ.get("IDENTITY_RSA_PRIVATE_KEY", """-----BEGIN RSA PRIVATE KEY-----
```

Comment:
hello secret key?

Reply:
lmao

Reply:
it's a dummy key big guy look at the environment substitution

Reply:
lel im blind